src/components/structural/Textures.js

```diff
@@ -0,0 +1,30 @@
+let TexturePack = [

```

Comment:
My original comment still stands. These should be hosted by us, not by a third party.